.gitignore

```diff
@@ -1,15 +1,12 @@
 .idea/
 .vscode/
-
 dist/
-
 .DS_Store
-/coverage.txt
-kubespray/
-/kubespray/generate_list.yml
+coverage.txt
+kubespray
 venv
```

Comment:
Adding a similar line `.venv` will be nice.
Sometimes I create a py virtual env by using `python3 -m venv .venv`, because hidden directory is a better choice in some perspectives.